measures/measures-schema.yaml

```diff
@@ -142,7 +142,7 @@ definitions:
       cpcPlusGroup:
         description: CPC+ group which the measure belongs
         type: string
-    required: [nationalQualityStrategyDomain, measureType, eMeasureId, nqfEMeasureId, nqfId, isHighPriority, isInverse, strata, primarySteward, measureSets, isRegistryMeasure]
+    required: [nationalQualityStrategyDomain, measureType, eMeasureId, nqfEMeasureId, nqfId, isHighPriority, isInverse, primarySteward, measureSets, isRegistryMeasure]
```

Comment:
This might be overkill - but an alternative to removing this validation would be to create another measure type so we can validate that strata and overallAlgorithm are present when the metricType is multiPerformanceRate - similar to [here](https://github.com/CMSgov/qpp-measures-data/pull/85/files#diff-cc1568c0c7b370d047afdf355f584710R37).

Reply:
(done!)